frontend/ui/otamanager.lua

```diff
@@ -73,12 +73,16 @@ function OTAManager:setOTAChannel(channel)
     G_reader_settings:saveSetting("ota_channel", channel)
 end
 
+function OTAManager:getZsyncFilename()
+    return self.zsync_template:format(model, channel)
```

Comment:
Probably best to use it as it was?
```suggestion
    return self.zsync_template:format(self:getOTAModel(), self:getOTAChannel())
```